### ISSUE TITLE
Add Continuous integration to project.

### ISSUE THREAD

**Issue by MorrisB--** [MorrisB--/MasterHash#25]
Consider adding CI after the Pre-Alpha is released.

**Comment by srajappa** [comment]
Jenkins is a good tool for CI :). I hope I provided an appropriate statement here regarding the issue.

**Comment by MorrisB--** [comment]
Yeah we use Jenkins at work, I was also considering Travis since it seems super popular on GitHub. In the meantime I have to learn a little more about CI before implementing it.

**Comment by patanric** [comment]
As far as I know Travis is ready to use, whereas Jenkins you have to install it somewhere and maintain it. Travis is free for open source projects.

**Comment by srajappa** [comment]
Oh is it ? I never knew that travis CI was free for projects that are open source.  In such case Travis is much better option in comparison to Jenkins; moreover one can add Travis build status badges to the README file.

**Comment by MorrisB--** [comment]
Do you have experience with Travis @srajappa ? I'll definitely start looking into it more

**Comment by srajappa** [comment]
As a DevOps engineer I use Jenkins daily, don't think Travis CI will be much different :). Haven't used Travis CI but from if you want me to explore and integrate it here, then i am all for help :)
